src/main/webapp/i18n/de/quizExercise.json

```diff
@@ -220,7 +220,17 @@
                 "title": "Kurzer Quiz Titel",
                 "questionTitle": "Fragen Titel",
                 "shortQuestionTitle": "Kurzer Fragen Titel"
-            }
+            },
+            "matchLetterCase": {
+                "title": "Groß- und Kleinschreibung beachten",
+                "explanation": "Wähle aus, ob die Groß- und Kleinschreibung der gegebenen Antworten bei der Auswertung eine Rolle spielt (Beispiel: Sollen 'Antwort' und 'antwort' als gleich bewertet werden?)"
+            },
+            "matchPercentage": {
+                "title": "Antworten genau abgleichen",
+                "explanationTitle": "Ähnlichkeit&nbsp;%",
```

Comment:
```suggestion
                "explanationTitle": "Ähnlichkeit",
```

Reply:
see below

Reply:
This was added explicitly by @jpbernius before ;-)